.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,37 @@
 <!--  Thanks for sending a pull request!  Here are some tips for you:
 1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label
+3. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+4. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
+5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
 -->
 
+**What type of PR is this?**
+> Uncomment only one, leave it on its own line:
+>
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+> /kind flake
+
 **What this PR does / why we need it**:
 
 **Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
 Fixes #
 
 **Special notes for your reviewer**:
 
-**Release note**:
-<!--  Write your release note:
-1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
-2. If no release note is required, just write "NONE".
+**Does this PR introduce a user-facing change?**:
+<!--  
+If no, just write "NONE".
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
```

Comment:
@guineveresaenger -- We use the following language in https://git.k8s.io/contributors/guide/release-notes.md:
> For pull requests that require additional action from users switching to the new release, include the string "action required" (case insensitive) in the release note:
>
>    ```release-note
>    action required: your release note here
>    ```

Does `action required: ` need to be prepended to the note or can it be anywhere? I don't know and was confused about where it should go, so if it needs to be prepended, let's be explicit about the language i.e., `s/include/prepend`

Reply:
Given @cjwagner's feedback on Slack, looks like the relnote only needs to contain the phrase and the location shouldn't matter, so I'm happy now :)